<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>484</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ingredientes</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color : #FFF;
	margin : 0px;
	padding : 0px;
}

QFrame#content {
	background-color : transparent;
	border: 1px solid #DADDE1;
	border-radius: 5px;
	min-width : 400px;
	min-height : 460px;
}

QLabel {
	font : 77 12px &quot;Arial&quot;;
	color : green;
}

QLineEdit, QComboBox, QTextEdit{
	background-color : transparent;
	border : 1px solid #DADDE1;
	border-radius : 5px;
	color : #616161;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="content">
     <property name="minimumSize">
      <size>
       <width>402</width>
       <height>462</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lbl_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>* Complete los campos para agregar los ingredientes.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tft_name">
        <property name="placeholderText">
         <string> * Ingrese el nombre del ingrediente</string>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="tft_cnt">
           <property name="placeholderText">
            <string> * Cantidad</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="tft_unit">
           <property name="placeholderText">
            <string> * Unidad de medida</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_add">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Mostrar productos</string>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	color : #6BA6FD;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
	background-color : #DBE9FF;
	padding-left : 10px;
	padding-right : 10px;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color : transparent;
	color : #6BA6FD;
	border: 1px solid #6BA6FD;
	border-radius: 5px;
}

</string>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea {
	border: none;
}

#scrollAreaWidgetContents {
	background-color : #FFFFFF;
}</string>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>454</width>
           <height>220</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="verticalLayout_task">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btn_cancel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Mostrar productos</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color : #FFFFFF;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
	background-color : red;
	padding-left : 20px;
	padding-right : 20px;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color : transparent;
	color : red;
	border: 1px solid red;
	border-radius: 5px;
}

</string>
           </property>
           <property name="text">
            <string>Cancelar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_accept">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Mostrar productos</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	color : #FFFFFF;
	border-style : solid;
	border-radius: 5px;
	min-height : 40px;
	font : 77 18px &quot;Arial&quot;;
	background-color : #42B72A;
	padding-left : 20px;
	padding-right : 20px;
}

QPushButton:hover {
	border: 1px solid #FFF;
	border-radius: 5px;
}

QPushButton:pressed {
	background-color : transparent;
	color : #42B72A;
	border: 1px solid #42B72A;
	border-radius: 5px;
}

</string>
           </property>
           <property name="text">
            <string>Aceptar</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
